pkg/util/crypto/certs.go

```diff
@@ -111,7 +111,9 @@ func LoadTlsConfigFromSecret(secret *corev1.Secret) (*tls.Config, error) {
 	rootCAs := x509.NewCertPool()
 	var tlsCert tls.Certificate
 
-	rootCAs.AppendCertsFromPEM(secret.Data[corev1.ServiceAccountRootCAKey])
+	if !rootCAs.AppendCertsFromPEM(secret.Data[corev1.ServiceAccountRootCAKey]) {
+		return nil, fmt.Errorf("failed to append ca certs")
```

Comment:
do we need to use pingcap/errors for stack trace ?

Reply:
Maybe we can add more info like this issue? https://github.com/pingcap/tidb-operator/issues/1936